### PR TITLE
feat(appeals): add greenbelt change page

### PR DESCRIPTION
**Pull request by AlexMoulin**
## Issue ticket number and link
https://pins-ds.atlassian.net/browse/A2-553
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/api/src/database/seed/data-test.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ const appealFactory = ({
 		},
 		applicationReference: generateLpaReference(),
 		address: { create: formattedAddress },
+		isGreenBelt: true,
```
Comment (james-hill-kc) [inline appeals/api/src/database/seed/data-test.js:117]: Should this be passed as a parameter or randomised, to ensure some test appeals have it set to true and others false?

File: appeals/web/src/server/appeals/appeal-details/green-belt/__tests__/__snapshots__/green-belt.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`green-belt GET /change should render the greenBelt change page when accessed from Appellant Case page with a back link to appellant case page 1`] = `
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/green-belt/__tests__/__snapshots__/green-belt.test.js.snap:3]: Just a thought, it's a little misleading that the name of the snapshot includes `with a back link` but the link isn't rendered in the snapshot. You could either render a snapshot that includes the whole page (although they do get quite large this way), or split the test so the snapshot can render just the back link (i.e. split the tests for `backLinkInnerHtml` and `mainInnerHtml`)

File: appeals/web/src/server/appeals/appeal-details/green-belt/green-belt.service.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+/**
+ *
+ * @param {import('got').Got} apiClient
+ * @param {string} appealId
+ * @param {string} updatedGreenBelt
+ * @returns {Promise<{}>}
+ */
+export function changeGreenBelt(apiClient, appealId, updatedGreenBelt) {
+	const formattedGreenBelt = convertFromYesNoToBoolean(updatedGreenBelt);
+	console.log(`Sending formattedGreenBelt to server. Value of: ${formattedGreenBelt}`);
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/green-belt/green-belt.service.js:12]: Was this left in intentionally? If so it should probably use `logger`

**Review by james-hill-kc — COMMENTED** [review]
Looks good, I've left a few comments

File: appeals/web/src/server/appeals/appeal-details/green-belt/green-belt.controller.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+		addNotificationBannerToSession(
+			session,
+			'changePage',
+			appealId,
+			undefined,
+			'Green belt status updated'
+		);
+
+		delete request.session.isGreenBelt;
+		console.log(origin);
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/green-belt/green-belt.controller.js:66]: Stray console.log here

**Review by james-hill-kc — APPROVED** [review]
LGTM! I noticed one stray console.log but nothing serious enough to block the PR

File: appeals/api/src/server/repositories/appeal.repository.js [hunk 5]
```diff
@@ -136,6 +137,7 @@ const updateAppealById = (
 			...(caseStartedDate && { caseStartedDate }),
 			...(caseValidDate && { caseValidDate }),
 			...(applicationReference && { applicationReference }),
+			isGreenBelt,
```
Comment (fabfab-kc) [inline appeals/api/src/server/repositories/appeal.repository.js:140]: Just unsure about this... I thought this field is in the appellant case and lpaq, and no longer in the main Appeal table?

**Review by fabfab-kc — DISMISSED** [review]
@AlexMoulin looks good to me, could you just check the single comment below?

**Comment by fabfab-kc** [comment]
Also, there is a security warning... I think @glpins has worked on fixing this type of issue...

**Review by james-hill-kc — APPROVED** [review]

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**